### PR TITLE
remove empty sub-samples

### PR DESCRIPTION
**Pull request by londumas**
Fix issue https://github.com/igmhub/picca/issues/306.
Remove empty sub-samples.

### CODE REVIEW

File: py/picca/test/test_cor.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def test_cor(self):
         self.send_cf1d()
         self.send_cf1d_cross()
 
-        self.send_cf_angl()
+        #self.send_cf_angl()
```
Comment (ngbusca) [inline py/picca/test/test_cor.py]: why is this test turned off?

**Comment by londumas** [comment]
@ngbusca, It was for tests, I have to understand what is happening.

**Comment by londumas** [comment]
@ngbusca, the issue is that in the master `py/picca/test/data/` some sub-samples are empty.
This PR removes them, then the unittest failes because there are not the same number of sub-samples. I would need to reproduce the /data/ files.
Or we can change nothing, we can live with some empty sub-samples.
What do you want?

**Comment by ngbusca** [comment]
I don't mind the empty subsamples...

**Comment by londumas** [comment]
ok, closing the PR and the ticket.

**Pull request closed without merging**